### ISSUE TITLE
The D2R save game folder could not be found here. As the message

### ISSUE THREAD

**Issue by wedqdas** [aeon0/botty#561]
The D2R save game folder could not be found here. As the message popped up, the bot execution and f9 setting key are not working. Do any angels know the solution? It might not be smooth because it's a translator. Thank you ㅠㅠ

**Comment by mgleed** [comment]
@wedqdas in params.ini:

```ini
[general]
name=Botty
message_api_type=generic_api
custom_message_hook=
max_game_length_s=380
max_consecutive_fails=0
randomize_runs=0
difficulty=hell
info_screenshots=1
loot_screenshots=0
discord_status_count=20
discord_status_condensed=0
saved_games_folder=
d2r_path=C:\Program Files (x86)\Diablo II Resurrected\D2R.exe  <<---- CHANGE THIS
```

**Comment by Kellen-Stuart** [comment]
> @wedqdas in params.ini:
> 
> ```ini
> [general]
> name=Botty
> message_api_type=generic_api
> custom_message_hook=
> max_game_length_s=380
> max_consecutive_fails=0
> randomize_runs=0
> difficulty=hell
> info_screenshots=1
> loot_screenshots=0
> discord_status_count=20
> discord_status_condensed=0
> saved_games_folder=
> d2r_path=C:\Program Files (x86)\Diablo II Resurrected\D2R.exe  <<---- CHANGE THIS
> ```

Don't you mean this?

> ```ini
> [general]
> name=Botty
> message_api_type=generic_api
> custom_message_hook=
> max_game_length_s=380
> max_consecutive_fails=0
> randomize_runs=0
> difficulty=hell
> info_screenshots=1
> loot_screenshots=0
> discord_status_count=20
> discord_status_condensed=0
> saved_games_folder=<<---- CHANGE THIS
> d2r_path=C:\Program Files (x86)\Diablo II Resurrected\D2R.exe  
> ```

**Comment by mgleed** [comment]
Addressed in #595
